packages/xforms-engine/src/lib/reactivity/validation/createValidation.ts

```diff
@@ -34,7 +35,8 @@ const engineViolationMessage = <Role extends ValidationTextRole>(
 ): Accessor<EngineViolationMessage<Role>> => {
 	const messageText = VALIDATION_TEXT[role];
 	const chunk = new TextChunk(context, 'literal', messageText);
-	const message = new TextRange('engine', role, [chunk]);
+	const evaluator = isEngineXPathEvaluator(context.evaluator) ? context.evaluator : null;
```

Comment:
@lognaturel Can a message display media? I think this message is from question type validations, contrains, ...

Reply:
I'm leaving validation messages without images because the spec doesn't mention media support for messages and it's out of scope (do we display red background? or red border?, etc.)